tests/api/test_adgroup.py

```diff
@@ -36,13 +36,13 @@ def test_get_adgroups_by_campaign_id(self):
     def test_post_and_get(self):
         """ Test for HTTP POST and GET
         """
-        url = url_for('api.adgroup.adgroups', campaign_id=self.new_campaign_id)
+        url = url_for('api.adgroup.adgroups')
         data = json.dumps(self.new_adgroup)
         response = self.client.post(url, data=data, content_type="application/json")
         assert_equal(response.status_code, 201)
         new = json.loads(response.data)["message"]
 
-        url = url_for('api.adgroup.adgroup', campaign_id=self.new_campaign_id, adgroup_id=new["id"])
+        url = url_for('api.adgroup.adgroup', adgroup_id=new["id"])
```

Comment:
yes, just figured out by reading the Flask docs. Sometimes RTFD does work :smile_cat: